packages/format-library/src/link/inline.js

```diff
@@ -195,28 +194,9 @@ function InlineLinkUI( {
 
 	const popoverAnchor = useAnchor( {
 		editableContentElement: contentRef.current,
-		settings,
+		settings: { ...settings, isActive },
```

Comment:
Is it better to add an `isActive` property instead of shoving it in the `settings` array? Probably... But I didn't want to have to update the docs to always support it if so.